aptos-move/aptos-gas-meter/src/lib.rs

```diff
@@ -7,7 +7,11 @@
 mod algebra;
 mod meter;
 mod traits;
+#[cfg(feature = "benchmark")]
```

Comment:
nit: we have a few places where we use the "bench" feature already, shall we follow the "convention"?